src/webhooks/interfaces/webhook.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface Webhook {
+  id: string;
+  event: string;
+  data: string;
```

Comment:
Should this be an object instead of a `string`?

Reply:
Thanks @maxdeviant ! I tried to address this and the two above issues, but let me know if anything still looks off 🙏